control-plane/go.mod

```diff
@@ -32,20 +32,20 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/stretchr/testify v1.8.4
-	go.uber.org/zap v1.24.0
+	go.uber.org/zap v1.25.0
 	golang.org/x/exp v0.0.0-20230817173708-d852ddb80c63
 	golang.org/x/text v0.14.0
 	golang.org/x/time v0.3.0
-	gomodules.xyz/jsonpatch/v2 v2.3.0
+	gomodules.xyz/jsonpatch/v2 v2.4.0
 	google.golang.org/grpc v1.58.3
 	google.golang.org/protobuf v1.33.0
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.26.12
-	k8s.io/apimachinery v0.26.12
-	k8s.io/client-go v0.26.12
+	k8s.io/api v0.28.9
+	k8s.io/apimachinery v0.28.9
+	k8s.io/client-go v0.28.9
```

Comment:
Bumping client-go here winds up introducing another swath of breaking changes in the control-plane. If it's okay with you given the size of this PR and the CVE resolutions tied to this, I think it's best to go ahead with this PR and follow up with making these versions consistent in the near future

Reply:
Oof, that’s surprising- agreed, as long as we think the mismatch is acceptable (I think it is since control-plane and cni are aligned) I’m totally good w/ stopping here for now 👍🏻